icicle/include/icicle/fields/host_math.h

```diff
@@ -293,6 +454,21 @@ namespace host_math {
       }
     }
   }
+  template <unsigned NLIMBS>
+  static constexpr void get_higher_with_slack(const storage<2 * NLIMBS>& xs, storage<NLIMBS>& out, unsigned slack_bits)
```

Comment:
how come this function with such a miserable name has no decription?

Reply:
lol good question, it's from the barret implementation so I also don't really know what's it for.. I can put it under the ifdef